prep-javascript-objects/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript Objects</title>
+</head>
+<body>
+  <script>
+  const hanSolo = {
+  firstName: 'Han',
+  lastName: 'Solo',
+  hobby: 'Piloting the Millennium Falcon',
+  age: 42,
+} </script>;
```

Comment:
You are missing more code! Make sure you are following the instructions. We told you that you should have been using bracket notation and dot notation to create more properties, while also adding console logs